config/env/default.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
   domain: process.env.DOMAIN || 'localhost:3000',
   supportEmail: 'support@trustroots.org', // TO-address for support requests
   profileMinimumLength: 140, // Require User.profile.description to be >=140 chars to send messages
+  // A message below longMessageMinimumLength will be 'short' in influxdb, otherwise 'long'
+  // TODO choose an expressive value
+  // Used in ./modules/messages/server/services/message-to-influx.server.service
+  longMessageMinimumLength: 150,
```

Comment:
@simison influxdb object is multiplicated across the config files, isn't it? Might be hard to update the value and sometimes it might be completely missing. Or do the config objects get filled their missing parameters from lower level configs? (i.e. uncommenting influxdb object in `config/env/local.js` will overwrite `config/env/default.js` or not?)